### PR TITLE
[MNT-24657] Add multiple selection support for people widget

### PR DESCRIPTION
**Pull request by nikita-web-ua**
**Please check if the PR fulfills these requirements**

> - [X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [X] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [X] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://hyland.atlassian.net/browse/MNT-24657

**What is the new behaviour?**

Multiple selection support for people widget was added

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services/src/lib/form/widgets/people/people.widget.html [hunk 1]
```diff
@@ -4,22 +4,37 @@
...
-               class="adf-input"
-               data-automation-id="adf-people-search-input"
-               type="text"
-               [id]="field.id"
-               [formControl]="searchTerm"
-               [placeholder]="field.placeholder"
-               [matAutocomplete]="auto"
-               (blur)="markAsTouched()"
-               [title]="field.tooltip">
+        <mat-chip-grid #chipGrid aria-label="People selection" data-automation-id="adf-people-widget-chip-list">
```
Comment (MichalKinas) [inline lib/process-services/src/lib/form/widgets/people/people.widget.html:7]: Aria-label should be localized

File: lib/process-services/src/lib/form/widgets/people/people.widget.spec.ts [hunk 2]
```diff
@@ -138,13 +191,63 @@ describe('PeopleWidgetComponent', () => {
...
         };
         widget.ngOnInit();
-
-        const involvedUser = fixture.debugElement.nativeElement.querySelector('input[data-automation-id="adf-people-search-input"]');
-
         fixture.detectChanges();
         await fixture.whenStable();
 
-        expect(involvedUser.value).toBe('John Doe');
+        const chip = await loader.getHarness(MatChipHarness.with({ selector: '[data-automation-id="adf-people-widget-chip-people-id"]' }));
```
Comment (MichalKinas) [inline lib/process-services/src/lib/form/widgets/people/people.widget.spec.ts:197]: This line is repeated couple of times, maybe we can move it to a function instead?

File: lib/process-services/src/lib/form/widgets/people/people.widget.ts [hunk 3]
```diff
@@ -144,11 +143,36 @@ export class PeopleWidgetComponent extends WidgetComponent implements OnInit {
...
+            this.field.value = user;
         }
+
+        this.peopleSelected.emit(user?.id || undefined);
+        this.input.nativeElement.value = '';
+        this.searchTerm.setValue('');
+    }
+
+    isUserAlreadySelected(user: LightUserRepresentation): boolean {
+        if (this.selectedUsers && this.selectedUsers.length > 0) {
```
Comment (MichalKinas) [inline lib/process-services/src/lib/form/widgets/people/people.widget.ts:171]:
```suggestion
        if (this.selectedUsers?.length > 0) {
```

File: lib/process-services/src/lib/form/widgets/people/people.widget.html (hunk 1 above)
Comment (nikita-web-ua) [reply lib/process-services/src/lib/form/widgets/people/people.widget.html]: good catch, fixed

File: lib/process-services/src/lib/form/widgets/people/people.widget.spec.ts (hunk 2 above)
Comment (nikita-web-ua) [reply lib/process-services/src/lib/form/widgets/people/people.widget.spec.ts]: done, ty

**Review by MichalKinas — APPROVED** [review]

**Review by swapnil-verma-gl — APPROVED** [review]

**Pull request merged**